profiles/car.lua

```diff
@@ -606,6 +606,16 @@ function way_function (way, result)
     end
   end
 
+  -- add driveability penalty to service roads
+  if properties.weight_name == 'duration' and highway == "service" then
+    if result.duration > 0 or result.forward_speed > 0 then
+      result.forward_weight_per_meter = result.forward_speed / 100;
```

Comment:
I am a bit confused. Isn't the weight supposed to be high, the moment we want to avoid a road?
Could you explain to me, why are we dividing by 100 here?